urlencode.py

```diff
@@ -21,18 +18,26 @@ def get_current_encoding(view, default='utf8'):
     return view_encoding
 
 
-def selections(view, default_to_all=True):
-    """Return all non-empty selections in view
-    If None, return entire view if default_to_all is True
-    """
-    regions = [r for r in view.sel() if not r.empty()]
+def updateSelection(self, edit, command):
```

Comment:
Instead of passing a string `command`, it would be much simpler if `updateSelection` took the encode/decode functions as an argument directly.

Reply:
You are right; I've been too lazy to check if Python had first-class functions ^^
I changed it.